composer.json

```diff
@@ -39,79 +39,74 @@
         "snapshots",
         "integration"
     ],
-    "autoload": {
-        "psr-4": { "Prooph\\": "src/" },
-        "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
-    },
-    "autoload-dev": {
-        "psr-4": { "ProophessorTest\\": "test/" }
-    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": "^7.0",
-        "symfony/symfony": "^3.1",
-        "doctrine/orm": "^2.5",
+        "php": "^7.1",
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/doctrine-cache-bundle": "^1.2",
-        "doctrine/doctrine-migrations-bundle": "^1.0",
+        "prooph/event-store-symfony-bundle": "dev-master",
+        "prooph/pdo-event-store": "^1.0",
+        "prooph/event-sourcing": "^5.2",
+        "prooph/event-store-bus-bridge": "^3.0",
+        "prooph/service-bus-symfony-bundle": "dev-master",
+        "symfony/asset": "^3.3",
+        "symfony/config": "^3.3",
```

Comment:
Not needed, the framework-bundle already requires this.

Reply:
k, removed.